luzer/tests/CMakeLists.txt

```diff
@@ -91,6 +91,13 @@ list(APPEND LUAJIT_FRIENDLY_MESSAGES
   "---- TRACE [0-9] stop -> loop"
   "---- TRACE [0-9] start [0-9]/[0-9] test_luajit_friendly.lua:[0-9]+"
   "---- TRACE [0-9] stop -> return"
+  "Total number of samples: 100"
+  "Total number of samples with recorded traces: 2 (2.0%)"
+  "Total number of samples with started traces: 2 (2.0%)"
```

Comment:
What is the value of information for starting and stopping traces?
I suppose the `recorded`/`aborted` metrics will be enough. The procentage for these metrics is confusing as well.